external/CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ list(APPEND EXTERNAL_INCLUDE_DIRS ${LIB_STATIC_APRUTIL1_INC_DIR})
 
 include(YamlLib)
 set(LIB_STATIC_YAML_LOC ${LIB_STATIC_YAML_LOC} PARENT_SCOPE)
+set(LIB_STATIC_YAML_INC_DIR ${LIB_STATIC_YAML_INC_DIR} PARENT_SCOPE)
```

Comment:
@vitaly-krugl I added these just for consistency with the other external cmake files. Agree that these YamlLib cmake changes can be reverted.